doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -211,28 +223,42 @@ Attributes of the ``uos_launcher`` specify the number of User VMs that the curre
   Specify the User VM memory size in Mbyte.
 
 ``gvt_args``:
-  GVT argument for the VM.
+  GVT argument for the VM. Recommendation is: 64 448 8. Leave it blank to disable the GVT.
 
 ``vbootloader``:
   Virtual bootloader type; currently only supports OVMF.
 
-``rootfs_dev``:
-  The device where User VM rootfs located.
-
-``rootfs_img``:
-  User VM rootfs image file including path.
+``cpu_sharing``:
+  Specify whether the binding pCPUs are sharing with other VMs.
 
-``console_type``:
-  Specify whether the User VM console is virtio or vUART; refer to :ref:`vuart_config` for details.
+``vuart0``:
+  Specify whether need device model to emulate vUART0(vCOM1); refer to :ref:`vuart_config` for details.
+  If it is Enable, means the vUART0 is emulated by device model;
```

Comment:
```suggestion
  If set to Enable, the vUART0 is emulated by the device model;
```

Reply:
Can you also enclose "Enable" in double backticks (so it appears as code)?